src/meta/src/barrier/progress.rs

```diff
@@ -338,6 +368,7 @@ impl CreateMviewProgressTracker {
     /// and then it will just report progress like newly created executors.
     fn recover_progress(
         states: HashMap<ActorId, BackfillState>,
+        backfill_order_state: Option<BackfillOrderState>,
```

Comment:
It seems that in this PR we haven't recovered the specified backfill order control yet in recovery. It so, the backfill_order_state here should always be None.

Given that this PR is already very large, we may remove this logic of passing the recovered backfill_order_map and include it in the PR that supports recovering the backfill order.

Reply:
https://github.com/risingwavelabs/risingwave/pull/20967/commits/2f8679b5652c3783d55d3ac19bf95d3e0fa890a1 removed recovery logic, so we no longer have this paramter